setup.py

```diff
@@ -56,5 +56,9 @@ def _dbt_sqlserver_version():
             "include/sqlserver/macros/**/**/*.sql",
         ]
     },
-    install_requires=["dbt-core~=0.18.0", "pyodbc>=4.0.27", "azure-identity>=1.4.0"],
+    install_requires=[
+        'dbt-core==0.19.0rc2',
```

Comment:
looks like @jtcohen6 was right that using the rc2 package from PyPI fixed the issue for us! it looks like we'll be able to make releases for all three dbt-msft packages within a day of of v0.19.0 coming out! pretty cool!

Reply:
@mikaelene def would appreciate building and pushing this branch to PyPI so we can have the same confidence about dbt-synapse. the dependency management from pinned GitHub releases is crazy-making!